utils/lstopo/lstopo-no-graphics.1in

```diff
@@ -447,6 +448,11 @@ If outputting to stdout and if colors are supported on the terminal,
 the output will be colorized.
 .
 .TP
+.B tikz or tex
+Output a LaTeX tikzpicture representation of the map that can be
+compiled with a LaTeX compiler to generate a standalone image.
```

Comment:
do we really need "to generate a standalone image" since the code may be inserted in the latex document?

Reply:
Changed for `Output a LaTeX tikzpicture representation of the map that can be compiled with a LaTeX compiler`